pandas/io/formats/html.py

```diff
@@ -138,10 +137,10 @@ def _write_cell(
         else:
             start_tag = "<{kind}>".format(kind=kind)
 
-        esc: Union[OrderedDict[str, str], Dict]
+        esc: Union[Dict[str, str], Dict]
```

Comment:
Arguably orthogonal but this is somewhat of a strange annotation - is it not always just `Dict[str, str]`?

Reply:
This line can be removed now as it is no longer required to silence `Incompatible types in assignment (expression has type "Dict[<nothing>, <nothing>]", variable has type "OrderedDict[str, str]")`.

Reply:
Sure - removed